tasks/oracle-db-11-12.yml

```diff
@@ -92,3 +90,45 @@
   tags:
     - skip_ansible_lint
   become: True
+
+- name: "Stop Oracle database"
+  shell: >
+    {{ oracledb.oracle_home }}/bin/dbshut {{ oracledb.oracle_home }}
+  tags:
+    - skip_ansible_lint
+  become: True
+
+- name: "Setup database port"
```

Comment:
this block should not be executed if we don't change default port?

Reply:
second, doesn't make sense to setup the port if it hasn't been changed in the first place, fixed